modules/account-lib/src/coin/near/transaction.ts

```diff
@@ -193,7 +268,7 @@ export class Transaction extends BaseTransaction {
       // txhash used to identify the transactions
       id: result.id || '',
       displayOrder,
-      outputAmount: result.actions[0].transfer.deposit.toString(),
+      outputAmount: '',
```

Comment:
why output here is empty?

Reply:
hmm so based on other coins, it should be initially zero, right?